Tasks/a0_my_stack.py

```diff
@@ -33,8 +33,13 @@ def peek(ind: int = 0):
         :param ind: index of element (count from the top)
         :return: peeked element
         """
+    try:
 
-    return stak[ind - 1]
+        s = stak[ind - 1]
+    except(IndexError):
```

Comment:
```suggestion
    except IndexError:
```
Более принятый на настоящий момент синтаксис